Makefile

```diff
@@ -101,23 +124,23 @@ cluster-sync-resources:
 	fi
 
 
-cluster-sync-handler: $(local_handler_manifest)
-	IMAGE_REGISTRY=localhost:$(shell ./cluster/cli.sh ports registry | tr -d '\r') \
+cluster-sync-handler: $(local_handler_manifest) $(CLI) $(KUBECTL)
+	IMAGE_REGISTRY=localhost:$(shell $(CLI) ports registry | tr -d '\r') \
 		make push-handler
-	./cluster/cli.sh ssh node01 'sudo docker pull $(LOCAL_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)'
+	$(SSH) node01 'sudo docker pull $(LOCAL_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)'
```

Comment:
why we need this?

Reply:
So we can change kubevirtci path just changing one line at the top of the Makefile.

Reply:
Let's drop it.